invenio_accounts/profiles/classes.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
```

Comment:
no, this was just the first thing that came to my mind because `utils.py` was already "taken" a level above...
a different name would indeed be better, what would you prefer?
`utils.py` (the `profiles` namespacing should make its purpose somewhat clear), `dicts.py`, or something else?

on a side note, i'm also not 100% happy with `profiles`, because it's also used for (the for now rudimentary) preferences, and might be used for other JSON fields...